keyboards/ergodox_ez/keymaps/hacker_dvorak/keymap.c

```diff
@@ -0,0 +1,797 @@
+        XXXXXXX, LCAG(KC_F2), LCAG(KC_F4), LCAG(KC_F6), LCAG(KC_F8), LCAG(KC_F10), LCAG(KC_F12),
+                 MEH(KC_F2),  MEH(KC_F4),  MEH(KC_F6),  MEH(KC_F8),  MEH(KC_F10),  MEH(KC_F12),
+        XXXXXXX, SGUI(KC_F2), SGUI(KC_F4), SGUI(KC_F6), SGUI(KC_F8), SGUI(KC_F10), SGUI(KC_F12),
+                              XXXXXXX,     XXXXXXX,     XXXXXXX,     XXXXXXX,      XXXXXXX,
+
+        // right thumb
+        XXXXXXX, XXXXXXX,
+        XXXXXXX,
+        XXXXXXX, XXXXXXX, XXXXXXX
+    ),
+
+    [MEH_FN_PLUS] = LAYOUT_ergodox(
+        // left hand
+        LCA(KC_F23),  LCA(KC_F21),  LCA(KC_F19),  LCA(KC_F17),  LCA(KC_F15),  LCA(KC_F13),  XXXXXXX,
+        LCAG(KC_F23), LCAG(KC_F21), LCAG(KC_F19), LCAG(KC_F17), LCAG(KC_F15), LCAG(KC_F13), XXXXXXX,
+        MEH(KC_F23),  MEH(KC_F21),  MEH(KC_F19),  MEH(KC_F17),  MEH(KC_F15),  MEH(KC_F13),
+        SGUI(KC_F23), SGUI(KC_F21), SGUI(KC_F19), SGUI(KC_F17), SGUI(KC_F15), SGUI(KC_F13), XXXXXXX,
+        XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,
+
+        // left thumb
+                 XXXXXXX, XXXXXXX,
+                          XXXXXXX,
+        XXXXXXX, XXXXXXX, XXXXXXX,
+
+        // right hand
+        XXXXXXX, LCA(KC_F14),  LCA(KC_F16),  LCA(KC_F18),  LCA(KC_F20),  LCA(KC_F22),  LCA(KC_F24),
+        XXXXXXX, LCAG(KC_F14), LCAG(KC_F16), LCAG(KC_F18), LCAG(KC_F20), LCAG(KC_F22), LCAG(KC_F24),
+                 MEH(KC_F14),  MEH(KC_F16),  MEH(KC_F18),  MEH(KC_F20),  MEH(KC_F22),  MEH(KC_F24),
+        XXXXXXX, SGUI(KC_F14), SGUI(KC_F16), SGUI(KC_F18), SGUI(KC_F20), SGUI(KC_F22), SGUI(KC_F24),
+                               XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      _______,
+
+        // right thumb
+        XXXXXXX, XXXXXXX,
+        XXXXXXX,
+        XXXXXXX, XXXXXXX, XXXXXXX
+    ),
+
+    [MAGICS] = LAYOUT_ergodox(
+        // left hand
+        EPRM,    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, _______,
+
+        // left thumb
+               XXXXXXX, XXXXXXX,
+                        XXXXXXX,
+        RESET, XXXXXXX, XXXXXXX,
+
+        // right hand
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, MS_GE,
+        MN_GUI,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, MU_GE,
+                 XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, MT_NKRO,
+        MY_GUI,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, MT_AG,
+                          XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+
+        // right thumb
+        XXXXXXX, XXXXXXX,
+        XXXXXXX,
+        XXXXXXX, MS_CC,   MU_CC
+    )
+};
+
+const uint16_t PROGMEM fn_actions[] = {
+    [1] = ACTION_LAYER_TAP_TOGGLE(1)
+};
+
+void matrix_init_user(void) {
+    rgblight_sethsv_noeeprom_green();
```

Comment:
I'm guessing that this doesn't work? 

That would be because it's called too early in the init process.  You'd be better off calling this in the first matrix scan.